docker-cloud/dockerid.md

```diff
@@ -2,15 +2,9 @@
 description: Using your DockerID to log in to Docker Cloud
 keywords:
 - one, two, three
-menu:
-  main:
-    parent: docker-cloud
-    weight: -99
-title: Docker ID and Settings
+title: Your Docker ID and Docker Cloud account
```

Comment:
Your Docker ID *is* your Docker Cloud account, so this isn't quite right. "Docker ID and Cloud Settings" is more correct.
(Again) this isn't reflected in the left nav, right?